static/app/utils/discover/fields.tsx

```diff
@@ -843,6 +845,15 @@ export function isAggregateEquation(field: string): boolean {
   return isEquation(field) && results !== null && results.length > 0;
 }
 
+export function isEquationColumn(column: Column): boolean {
+  // Any isn't allowed in arithmetic
+  if (column.kind === 'function' && column.function[0] === 'any') {
+    return false;
+  }
+  const columnType = getColumnType(column);
+  return columnType === 'number' || columnType === 'integer' || columnType === 'duration';
```

Comment:
With the upcoming mobile vitals, should we include `'percentage'` as an allowed type?

Reply:
+1, but will have to be in 2 separate PRs so I can do the backend changes to allowlist them first